sql/core/src/test/resources/sql-tests/inputs/pgSQL/timestamp.sql

```diff
@@ -0,0 +1,247 @@
+
+SELECT '' AS `16`, d1 FROM TIMESTAMP_TBL
+   WHERE d1 <= timestamp '1997-01-02';
+
+SELECT '' AS `49`, d1 FROM TIMESTAMP_TBL
+   WHERE d1 >= timestamp '1997-01-02';
+
+-- [SPARK-28425] Add more Date/Time Operators
+-- SELECT '' AS `54`, d1 - timestamp '1997-01-02' AS diff
+--    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
+
+SELECT '' AS date_trunc_week, date_trunc( 'week', timestamp '2004-02-29 15:44:17.71393' ) AS week_trunc;
+
+-- [SPARK-28425] Add more Date/Time Operators
+-- Test casting within a BETWEEN qualifier
+-- SELECT '' AS `54`, d1 - timestamp '1997-01-02' AS diff
+--   FROM TIMESTAMP_TBL
+--   WHERE d1 BETWEEN timestamp '1902-01-01'
+--    AND timestamp '2038-01-01';
+
+-- [SPARK-28420] Date/Time Functions: date_part
+-- SELECT '' AS "54", d1 as "timestamp",
+--    date_part( 'year', d1) AS year, date_part( 'month', d1) AS month,
+--    date_part( 'day', d1) AS day, date_part( 'hour', d1) AS hour,
+--    date_part( 'minute', d1) AS minute, date_part( 'second', d1) AS second
+--    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
+
+-- SELECT '' AS "54", d1 as "timestamp",
+--    date_part( 'quarter', d1) AS quarter, date_part( 'msec', d1) AS msec,
+--    date_part( 'usec', d1) AS usec
+--    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
+
+-- SELECT '' AS "54", d1 as "timestamp",
+--    date_part( 'isoyear', d1) AS isoyear, date_part( 'week', d1) AS week,
+--    date_part( 'dow', d1) AS dow
+--    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
+
+-- [SPARK-28137] Data Type Formatting Functions
+-- TO_CHAR()
+-- SELECT '' AS to_char_1, to_char(d1, 'DAY Day day DY Dy dy MONTH Month month RM MON Mon mon')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_2, to_char(d1, 'FMDAY FMDay FMday FMMONTH FMMonth FMmonth FMRM')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_3, to_char(d1, 'Y,YYY YYYY YYY YY Y CC Q MM WW DDD DD D J')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_4, to_char(d1, 'FMY,YYY FMYYYY FMYYY FMYY FMY FMCC FMQ FMMM FMWW FMDDD FMDD FMD FMJ')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_5, to_char(d1, 'HH HH12 HH24 MI SS SSSS')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_6, to_char(d1, E'"HH:MI:SS is" HH:MI:SS "\\"text between quote marks\\""')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_7, to_char(d1, 'HH24--text--MI--text--SS')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_8, to_char(d1, 'YYYYTH YYYYth Jth')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_9, to_char(d1, 'YYYY A.D. YYYY a.d. YYYY bc HH:MI:SS P.M. HH:MI:SS p.m. HH:MI:SS pm')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_10, to_char(d1, 'IYYY IYY IY I IW IDDD ID')
+--    FROM TIMESTAMP_TBL;
+
+-- SELECT '' AS to_char_11, to_char(d1, 'FMIYYY FMIYY FMIY FMI FMIW FMIDDD FMID')
+--    FROM TIMESTAMP_TBL;
+
+-- timestamp numeric fields constructor
+-- SELECT make_timestamp(2014,12,28,6,30,45.887);
```

Comment:
Could you file a JIRA for this `make_timestamp` function since this is not SPARK-28137 and SPARK-28425.

Reply:
And, please add the new JIRA ID as a comment.

Reply:
OK. Add [SPARK-28432](https://issues.apache.org/jira/browse/SPARK-28432).